src/Steps/index.tsx

```diff
@@ -32,20 +36,31 @@ export default function Steps(props: StepsProps) {
       }
     }
 
-    return Array.from(dotSet);
+    return {
+      marksValue: uniqueMarksValue,
+      stepDots: Array.from(dotSet),
+    };
   }, [min, max, step, dots, marks]);
 
   return (
     <div className={`${prefixCls}-step`}>
-      {stepDots.map((dotValue) => (
-        <Dot
-          prefixCls={prefixCls}
-          key={dotValue}
-          value={dotValue}
-          style={style}
-          activeStyle={activeStyle}
-        />
-      ))}
+      {stepDots.map((dotValue) => {
+        // Check whether it is a marks dot
+        const isMarksDot = marksValue.indexOf(dotValue) >= 0;
+
+        return (
+          <Dot
+            prefixCls={prefixCls}
+            className={classNames({
+              [`${prefixCls}-marks-dot`]: isMarksDot,
```

Comment:
Add css for `${prefixCls}-marks-dot`  in here https://github.com/react-component/slider/blob/master/assets/index.less